migrations/202006031556-CreateMessageTable.js

```diff
@@ -21,10 +21,9 @@ module.exports = {
                 allowNull: false,
                 type: Sequelize.STRING
             },
-            email: {
-                type: Sequelize.STRING,
-                allowNull: false,
-                defaultValue: "hello"
+            user_id: {
```

Comment:
similar to our conversation last week, we probably want to do this as a migration. so rather than changing the original, we create a new migration that _removes_ the email column, and adds the `user_id` column, and maybe even writes good data to the `user_id` column based on `email` while it's running